components/automate-ui/src/app/modules/user/user-details/user-details.component.ts

```diff
@@ -37,6 +37,7 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
   private isDestroyed = new Subject<boolean>();
   private deletingUser = false;
   public updatingUser = false;
+  public preUrlUserId = '';
```

Comment:
looks like we can drop this variable

Reply:
@vinay033 can you take this unused variable out?

Reply:
yes, I removed.

Reply:
hmm, but its still showing in the files changed??